streaming/index.js

```diff
@@ -183,6 +184,73 @@ const startServer = async () => {
   const redisSubscribeClient = await redisUrlToClient(redisParams, redisUrl);
   const redisClient = await redisUrlToClient(redisParams, redisUrl);
 
+  // Collect metrics from Node.js
+  metrics.collectDefaultMetrics();
```

Comment:
We can optionally prefix all these default metrics with a prefix? But we've previously not prefixed the prometheus metrics, so it's probably safer to leave them unprefixed for 4.2.0, then add a prefix in 5.0.0 or 4.3.0 or something.